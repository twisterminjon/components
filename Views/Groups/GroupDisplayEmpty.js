import React from 'react'
import PropTypes from 'prop-types'
import GroupHeader from './GroupHeader'
import GroupFooter from './GroupFooter'
import './GroupDisplay.css'

GroupDisplayEmpty.propTypes = {
  name: PropTypes.string.isRequired,
  onGoBack: PropTypes.func.isRequired,
}

export default function GroupDisplayEmpty({ name, onGoBack }) {
  return (
    <div className="groupdisplay-wrap">
      <GroupHeader name={name} onClick={onGoBack} />
      <span className="groupdisplay-list groupdisplay-text">No staff members have been added to this group.</span>
      <GroupFooter disabled={true} />
    </div>
  )
}
