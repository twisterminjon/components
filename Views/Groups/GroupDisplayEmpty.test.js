import React from 'react'
import GroupDisplayEmpty from './GroupDisplayEmpty'

describe('GroupDisplayEmpty', () => {
  const mockFun = jest.fn()

  it('matches the snapshot', () => {
    const wrapper = window.shallow(<GroupDisplayEmpty name="The BeeGees" onGoBack={mockFun} />)
    expect(wrapper).toMatchSnapshot()
  })
})
