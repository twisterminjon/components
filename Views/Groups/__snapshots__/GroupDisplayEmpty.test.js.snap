// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GroupDisplayEmpty matches the snapshot 1`] = `
<div
  className="groupdisplay-wrap"
>
  <GroupHeader
    name="The BeeGees"
    onClick={[MockFunction]}
  />
  <span
    className="groupdisplay-list groupdisplay-text"
  >
    No staff members have been added to this group.
  </span>
  <GroupFooter
    disabled={true}
    onClick={[Function]}
  />
</div>
`;
